lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -104,6 +105,14 @@
     name: default
     state: absent
     cluster: new_cluster
+
+# Checking whether service is deployed
+- ecs_service:
+    state: deployed
+    name: console-test-service
+    cluster: new_cluster
+    task_definition: new_cluster-task:2"
+    desired_count: 1
```

Comment:
Good catch. I think this can be removed, since the question 'Whether deployment was successful or not?' just needs to verify against already configured `desired_count`